IHP/IDE/SchemaDesigner/View/Columns/Edit.hs

```diff
@@ -113,6 +113,9 @@ typeSelector postgresType enumNames = [hsx|
             {option selected "Time" "Time"}
             {option selected "SERIAL" "Serial"}
             {option selected "BIGSERIAL" "Bigserial"}
+            {option selected "POINT" "Point"}
```

Comment:
While testing I got this error in the generated haskell code:

Not in scope: type constructor or class ‘Point’